static/api/apidocs-4.19/apis/createVMSchedule.html

```diff
@@ -43,7 +43,7 @@ <h2>Request parameters</h2>
                                             <td style="width:200px;"><strong>Parameter Name</strong></td><td style="width:500px;">Description</td><td style="width:180px;">Required</td>
                                         </tr>
                                         <tr>
-                                            <td style="width:200px;"><strong>action</strong></td><td style="width:500px;"><strong>Action to take on the VM (start/stop/restart/force_stop/force_reboot).</strong></td><td style="width:180px;"><strong>true</strong></td>
+                                            <td style="width:200px;"><strong>action</strong></td><td style="width:500px;"><strong>Action to take on the VM (start/stop/reboot/force_stop/force_reboot).</strong></td><td style="width:180px;"><strong>true</strong></td>
```

Comment:
I believe to have this change in future releases as well, you may want to raise a PR against the apache/cloudstack repo - https://github.com/apache/cloudstack/blob/main/api/src/main/java/org/apache/cloudstack/api/command/user/vm/CreateVMScheduleCmd.java#L68-L71 
which is where this documentation is originally created.

Reply:
Done https://github.com/apache/cloudstack/pull/9226